packages/projects-docs/pages/learn/sandboxes/overview.mdx

```diff
@@ -35,5 +37,15 @@ If your project requires advanced configuration, try using a Cloud Sandbox inste
        - Branching
        - Git tooling 
        - Full integration with GitHub
+       
+        ## What is a Synced Sandbox?
+
+        With CodeSandbox, you can import any GitHub repository, subfolder or branch as a synced cloud sandbox. 
+        
+        An imported sandbox will automatically stay in sync with the GitHub repository; this means that if you make a commit to GitHub, it will reflect immediately in the sandbox.
```

Comment:
```suggestion
A Synced sandbox is a Sandbox that stays up to date with the latest changes you have pushed to a GitHub project using your `.git` file.

Their main advantage is that they allow you to create sandboxes from specific sub-folders within your GitHub projects without importing the whole repo. 

They automatically stay in sync with the GitHub repository folder; this means that if you make a commit to GitHub, it will reflect immediately in the sandbox.
```